### ISSUE TITLE
Option to output as large number as scientific notation

### ISSUE THREAD

**Issue by jowinjestine** [tompollard/tableone#163]
I would like an option to return large number in scientific notation

**Comment by tompollard** [comment]
@jowinjestine thanks for the suggestion! do you have any thoughts on how you would want to specify the notation in the arguments? e.g. is something like `scientific_notation=True` sufficient?

**Comment by jowinjestine** [comment]
Yes that definitely works, having an option to pass parameter such as scientific notation= True or False, 
I would say to keep it False as default
